tasks.py

```diff
@@ -92,7 +92,7 @@ def fix_black(ctx: Context, args: str = "."):
 def fix_ruff(ctx: Context, args: str = "."):
     """fix all ruff rules"""
     args = args or "."  # needed for hatch script
-    ctx.run(f"ruff --fix {args}", pty=use_pty)
+    ctx.run(f"ruff check --fix {args}", pty=use_pty)
```

Comment:
Ah! https://github.com/astral-sh/ruff/blob/main/CHANGELOG.md#cli

Reply:
Yep ... so minor I did not took time to mention it, but obviously I should have to spare your time ^^